eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/impl/AlertEagleStorePlugin.java

```diff
@@ -62,10 +60,9 @@ public void onAlert(AlertStreamEvent event) throws Exception {
         if (eventList == null || eventList.isEmpty()) {
             return;
         }
-        List<AlertPublishEvent> alertEvents = new ArrayList<>();
-        for (AlertStreamEvent e : eventList) {
-            alertEvents.add(AlertPublishEvent.createAlertPublishEvent(e));
-        }
+        List<AlertPublishEvent> alertEvents = eventList.stream().map(
```

Comment:
I have created a non-professional benchmark [test](https://github.com/asdf2014/yuzhouwan/blob/master/yuzhouwan-hacker/src/main/java/com/yuzhouwan/hacker/lambda/StreamPerformance.java#L19). After adding `-Xmx2G -Xms2G -Xmn1G -XX:+AlwaysPreTouch` params for reducing the impact from jvm resources, then found the performance gap is not very large, and the performance of `lambda stream` is better than `simple for loop` after the data becomes larger.